html/apps/applist/index.html

```diff
@@ -71,6 +71,7 @@
                   <div class="hidden">
                     <div class="caption">
                       <div class="centered">
+                        <p><a href='../../apps/{{ app.name }}'>{{ app.headline }}</a></p>
```

Comment:
It's okay to just href this link to `"/apps/{{ app.name }}"` instead of `../../`